source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: "Home Assistant Companion for Mac"
+description: "Integrate your Mac into Home Assistant."
+date: 2020-09-18 00:00:00
```

Comment:
This will hide it from the preview, that's annoying. Let's temporarily make it visible.
```suggestion
date: 2020-09-16 02:00:00
```